package/crds/apigatewayv2.aws.crossplane.io_deployments.yaml

```diff
@@ -0,0 +1,204 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.0
+  creationTimestamp: null
+  name: deployments.apigatewayv2.aws.crossplane.io
+spec:
+  group: apigatewayv2.aws.crossplane.io
+  names:
+    categories:
+    - crossplane
+    - managed
+    - aws
+    kind: Deployment
```

Comment:
Could this create a problem when listing deployments (from k8s `v1` api)? I can definitely imagine scripts naively wrapping around `kubectl get deployments` without filtering by api version.

Reply:
Kubernetes kinds take precedence. So, users would have to call it like `kubectl get deployments.apigatewayv2` or like others `kubectl get managed` or `kubectl get aws` etc.